login/src/androidMain/kotlin/org/dhis2/mobile/login/main/data/LoginRepositoryImpl.kt

```diff
@@ -220,9 +250,17 @@ class LoginRepositoryImpl(
         }
     }
 
+    override fun updateBiometricsPermissions(granted: Boolean) {
```

Comment:
shouldn't this operation be in a IO thread?: Due that it's used in `CredentialsViewModel` in the main dispatcher.